src/reconcile/rec_row.c

```diff
@@ -770,6 +794,8 @@ __wt_rec_row_leaf(
           __wt_txn_visible_all(session, tw.stop_txn, tw.stop_ts))
             upd = &upd_tombstone;
 
+        __rec_simplify_time_window(session, &tw);
```

Comment:
This is a copy of the `upd_select` time window - is the aggregated time window calculated based on this time window or the one in `upd_select`?

Reply:
Its based off this one.

Reply:
The outstanding test failure looks like maybe we are making a decision somewhere for aggregated time windows based on a different time window:

```
newest durable: (0, 154086)/(0, 330217) oldest start: (0, 0)/0 newest stop (4294967295, 4294967295)/18446744073709551605 failed verification with start durable timestamp of (0, 275245), greater than the parent's newest durable start timestamp of (0, 154086)
```